veashi-scanner/globals.css

```
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,400;9..40,500;9..40,600;9..40,700&family=JetBrains+Mono:wght@400;500;600&display=swap');
+@import "@kleros/ui-components-library/theme.css";
+@import "tailwindcss";
+@source "./node_modules/@kleros/ui-components-library";
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)